game_calls.cpp

```diff
@@ -1,14 +1,25 @@
-#include "game_calls.h"
+#include <algorithm>
 #include <iostream>
 #include <limits>
 #include <ctime>
+#include "game_calls.h"
+
+Code::Code()
+{
+	// Easier to write! Arguments: number of elements, value of each element
+	code = std::vector<int>(4, 0);
+}
+
+Code::~Code() = default;		// .clear() will be called by ~vector()
 
 void Code::UpdateGuess()
 {
-	srand(time(NULL));
+	// Try using std::random instead
+	//srand(time(NULL));		// Compiler throws a warning here; so explicitly cast it
+	srand(static_cast<unsigned int>(time(NULL)));
```

Comment:
So this forces "time(NULL)" to become an int?

Reply:
*Unsigned* int, yes. Not "forces", but "casts": it may do bit shifting etc behind the scenes.